ecs/fargate/main.tf

```diff
@@ -26,6 +26,42 @@ resource "aws_iam_role_policy" "service_permissions" {
   policy = "${data.aws_iam_policy_document.service_permissions.json}"
 }
 
+# ------------------------------------------------------------------------------
+# IAM - Task execution role, needed to pull ECR images etc.
+# ------------------------------------------------------------------------------
+resource "aws_iam_role" "execution" {
+  name               = "${var.prefix}-task-execution-role"
+  assume_role_policy = "${data.aws_iam_policy_document.task_assume.json}"
+}
+
+resource "aws_iam_role_policy" "task_execution" {
+  name   = "${var.prefix}-task-execution"
+  role   = "${aws_iam_role.execution.id}"
+  policy = <<EOF
```

Comment:
Should we put this policy as a `data "aws_iam_policy_document"` in policies.tf to be consistent?